lua/entities/gmod_tardis/modules/sh_destination.lua

```diff
@@ -110,6 +110,41 @@ TARDIS:AddKeyBind("destination-snaptofloor",{
 })
 
 if SERVER then
+    function ENT:SetDestination(pos, ang)
+        if self:CallCommonHook("CanChangeDestination", pos, ang) == false then
+            return false
+        end
+
+        if not isvector(pos) or not isangle(ang) then
+            self:SetData("destination_pos",nil,true)
+            self:SetData("destination_ang",nil,true)
+            return false
```

Comment:
Should we be returning false here? It is unclear if the change failed or if it just cleared the destination from the return atm

Reply:
It returns whether destination is currently set rather than the success of the action
Seems more convenient, I'd prefer to keep it this way
Situations in which we clear the destination do not cause any notifications / output, so it's okay

Reply:
Okay, mainly just wanted to check this was intentional and wasn't going to mess up any callers